### ISSUE TITLE
Calling kss-node will always run init

### ISSUE THREAD

**Issue by queenvictoria** [kss-node/kss-node#192]
For some reason when I built this this morning the command will run `init` regardless of whether init is specified or not.

```bash
$ ./node_modules/kss/bin/kss-node --template=src/styleguide/ --source=src/less/ --destination=dist/styleguide
Creating a new style guide template...
Error! This folder already exists: /srv/www/example.com/local/styleguide-template
```

After some debugging I think that `yargs.argv` is now being overridden by the `generatorConfig` builder. Commenting out `options('init')` means that the initialise doesn't run except when called.

Perhaps there is a recent change (last 7 days) to yargs?

```javascript
/**
 * Generator configruation.
 * @type {Object}
 */
generatorConfig = yargs
  .usage('Usage: kss-node <source> [destination] [options]')
  .wrap(80)
  .strict()

  // .options('init', {
  //   alias     : 'i',
  //   describe  : 'Create a new style guide template to customize',
  //   default   : 'styleguide-template'
  // })
```

**Comment by JohnAlbin** [comment]
Yep. Fixed in b749148a209d5c8741d7fb7836c2ad02134d57de.

Oh, crap. 2.0.2 is the latest official release and it doesn't contain that fix.

**Comment by JohnAlbin** [comment]
Oh, wait. 2.0.2 just uses yargs 1.x. It is yargs 3.5 that breaks kss-node and it was only allowed on the master branch post-2.0.2.
